cmd/server.go

```diff
@@ -20,7 +20,7 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/mitchellh/go-homedir"
+	homedir "github.com/mitchellh/go-homedir"
```

Comment:
Why do you have all these named imports? What tool is adding them back?

Reply:
Can you remove them please unless there is a requirement somewhere that we need them now.l

Reply:
I assumed it was the linter, I didn't add them explicitly. I'm just using `make gometalint`.

Reply:
The linter won't add anything, just fail

Reply:
Actually, it seems like the issue was using `goreturns` to format. I switched to using `goimports` and it stopped doing it.